osgp/platform/osgp-adapter-ws-core/src/main/java/org/opensmartgridplatform/adapter/ws/core/application/config/NotificationClientConfig.java

```diff
@@ -1,126 +1,29 @@
-
-    @Value("${web.service.truststore.password:123456}")
-    private String webserviceTruststorePassword;
-
-    @Value("${web.service.truststore.type:jks}")
-    private String webserviceTruststoreType;
-
-    @Value("${web.service.keystore.location:/etc/ssl/certs}")
-    private String webserviceKeystoreLocation;
-
-    @Value("${web.service.keystore.password:1234}")
-    private String webserviceKeystorePassword;
-
-    @Value("${web.service.keystore.type:pkcs12}")
-    private String webserviceKeystoreType;
-
-    @Bean
-    public SaajSoapMessageFactory messageFactory() {
-        return new SaajSoapMessageFactory();
-    }
-
-    @Bean
-    public KeyStoreFactoryBean webServiceTrustStoreFactory() {
-        final KeyStoreFactoryBean factory = new KeyStoreFactoryBean();
-        factory.setType(this.webserviceTruststoreType);
-        factory.setLocation(new FileSystemResource(this.webserviceTruststoreLocation));
-        factory.setPassword(this.webserviceTruststorePassword);
-        return factory;
-    }
-
-    public Jaxb2Marshaller notificationSenderMarshaller() {
-        final Jaxb2Marshaller marshaller = new Jaxb2Marshaller();
-        marshaller.setContextPath(this.marshallerContextPathNotification);
-        return marshaller;
-    }
-
-    @Bean
-    public SendNotificationServiceClient sendNotificationServiceClient() {
-        return new SendNotificationServiceClient(
-                this.createWebServiceTemplateFactory(this.notificationSenderMarshaller()),
-                this.webserviceNotificationOrganisation, this.webserviceNotificationUsername);
-    }
-
-    @Bean
-    public DefaultWebServiceTemplateFactory createWebServiceTemplateFactory(final Jaxb2Marshaller marshaller) {
-        final DefaultWebServiceTemplateFactory.Builder builder = new DefaultWebServiceTemplateFactory.Builder();
-        builder.setMarshaller(marshaller).setMessageFactory(this.messageFactory())
-                .setTargetUri(this.webserviceNotificationUrl).setMaxConnectionsPerRoute(this.maxConnectionsPerRoute)
-                .setMaxConnectionsTotal(this.maxConnectionsTotal).setApplicationName(this.applicationName)
-                .setSecurityEnabled(this.webserviceNotificationSecurityEnabled);
-        if (this.webserviceNotificationSecurityEnabled) {
-            builder.setKeyStoreType(this.webserviceKeystoreType).setKeyStoreLocation(this.webserviceKeystoreLocation)
-                    .setKeyStorePassword(this.webserviceKeystorePassword)
-                    .setTrustStoreFactory(this.webServiceTrustStoreFactory());
-        }
-
-        return builder.build();
-    }
+@PropertySource("classpath:osgp-adapter-ws-core.properties")
+@PropertySource(value = "file:${osgp/Global/config}", ignoreResourceNotFound = true)
+@PropertySource(value = "file:${osgp/AdapterWsCore/config}", ignoreResourceNotFound = true)
+public class NotificationClientConfig extends PublicLightingNotificationClientConfigBase {
```

Comment:
Consider renaming class to CoreNotificationClientConfig

Reply:
Done